### ISSUE TITLE
Lazy evaluation of schema

### ISSUE THREAD

**Issue by rossgray** [mvantellingen/python-zeep#678]
Hi,

I am working with a very large WSDL that has a large number of complex types and hence it takes a long time to parse. I'm only using a small subset of the operations defined in the WSDL so would like to avoid parsing the whole WSDL and its many XSDs. Is there any way to get the types needed in a more lazy manner, rather than parsing them all? Or do you have any other suggestions?

Thanks!

**Comment by mvantellingen** [comment]
Did you enable caching btw?

**Comment by rossgray** [comment]
No, I couldn't find a way of caching the WSDL since I couldn't pickle it. In the end I am just storing the WSDL document in memory, which isn't ideal since it cannot be shared across processes.
